src/frontend/src/pages/index.js

```diff
@@ -42,14 +45,54 @@ export default function IndexPage() {
     getPosts(numPages);
   }, [telescopeUrl]);
 
+  useEffect(() => {
+    async function getPostsCount() {
+      try {
+        const res = await fetch(`https://dev.telescope.cdot.systems/posts`);
+        // const res = await fetch(`${telescopeUrl}/posts`);
+        if (!res.ok) {
+          throw new Error(res.statusText);
+        }
+        return res.headers.get('x-total-count');
```

Comment:
I'm not sure if this could be a problem. The way we do it in the backend is like this:
`res.set('X-Total-Count', postsCount);`
If yours fails it could because `get` is case-sensitive?

Reply:
We both tested ytd and have verified it is case-insensitive